### PR TITLE
added MovieController, MovieSessionController and AuthenticationController and Dto for Movie and MovieSession

### PR DESCRIPTION
**Pull request by KostiantynVorobiov**

### CODE REVIEW

**Review by aleksandr-hrankin — APPROVED** [review]

File: src/main/java/com/cinema/controller/AuthenticationController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public String addMovie(@RequestBody UserRequestDto userRequestDto) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/controller/AuthenticationController.java]: addMovie?

File: src/main/java/com/cinema/controller/MovieController.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    private final MovieDtoMapper movieDtoMapper;
+
+    @Autowired
+    public MovieController(MovieService movieService, MovieDtoMapper movieDtoMapper) {
+        this.movieService = movieService;
+        this.movieDtoMapper = movieDtoMapper;
+    }
+
+    @GetMapping
+    public List<MovieResponseDto> getAllMovie() {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/controller/MovieController.java]:
```suggestion
    public List<MovieResponseDto> getAll() {
```

File: src/main/java/com/cinema/controller/MovieController.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        return movieService.getAll().stream()
+                .map(movieDtoMapper::convertToResponseDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    public String addMovie(@RequestBody MovieRequestDto movieRequestDto) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/controller/MovieController.java]:
```suggestion
    public String add(@RequestBody MovieRequestDto movieRequestDto) {
```

File: src/main/java/com/cinema/dao/MovieDao.java [hunk 4]
```diff
@@ -6,5 +6,7 @@
 public interface MovieDao {
     Movie add(Movie movie);
 
+    public Movie getById(Long id);
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/dao/MovieDao.java]:
```suggestion
    public Movie get(Long id);
```

File: src/main/java/com/cinema/dao/CinemaHallDao.java [hunk 5]
```diff
@@ -6,5 +6,7 @@
 public interface CinemaHallDao {
     CinemaHall add(CinemaHall cinemaHall);
 
+    CinemaHall getById(Long id);
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/dao/CinemaHallDao.java]: same

File: src/main/java/com/cinema/dao/impl/CinemaHallDaoImpl.java [hunk 6]
```diff
@@ -47,6 +47,16 @@ public CinemaHall add(CinemaHall cinemaHall) {
         }
     }
 
+    @Override
+    public CinemaHall getById(Long id) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/dao/impl/CinemaHallDaoImpl.java]:
```suggestion
    public CinemaHall get(Long id) {
```

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java [hunk 7]
```diff
@@ -46,6 +46,16 @@ public Movie add(Movie movie) {
         }
     }
 
+    @Override
+    public Movie getById(Long id) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: same

File: src/main/java/com/cinema/model/dto/MovieSessionResponseDto.java [hunk 8]
```diff
@@ -0,0 +1,11 @@
+package com.cinema.model.dto;
+
+import lombok.Data;
+
+@Data
+public class MovieSessionResponseDto {
+    private Long id;
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/model/dto/MovieSessionResponseDto.java:7]: let's add movieId

File: src/main/resources/db.properties [hunk 9]
```diff
@@ -6,5 +6,6 @@ db.password=12345
 
 # Hibernate properties
 hibernate.show_sql=true
-hibernate.hbm2ddl.auto=create-drop
+//hibernate.hbm2ddl.auto=create-drop
+hibernate.hbm2ddl.auto=update
```
Comment (dmytro-hryhoruk) [inline src/main/resources/db.properties]: let's remove comment

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**